src/scripts/manually_filter_reddit_comments.py

```diff
@@ -0,0 +1,140 @@
+"""Script for manually filtering a list of reddit comments."""
+
+import pandas as pd
+from datasets import Dataset, load_dataset
+import nltk
+import click
+from pathlib import Path
+from nltk.tokenize import sent_tokenize
+import itertools as it
+import re
+
+
+# Download the sentence splitter model
+nltk.download("punkt", quiet=True)
+
+
+@click.command("Starts the process of manually filter reddit comments.")
+@click.option("--output-dir", type=Path, required=True)
+@click.option("--n", type=int, default=1000)
```

Comment:
I would include more info here. I'm guessing this is the number of samples to annotate:

```suggestion
@click.option("--num-samples=", "-n", type=int, default=1000, help="The number of samples to annotate")
```